### PR TITLE
Composite layer to transfer positionOffset and modelMatrix

### PR DESCRIPTION
**Pull request by shaojingli**
Transfer positionOffset and modelMatrix props properly to their underlying layers

@Pessimistress @ibgreen

### CODE REVIEW

**Comment by Pessimistress** [comment]
Looks like `positionOffset` has been renamed to `positionOrigin`. The base layer doc needs to be updated.

**Comment by shaojingli** [comment]
> Looks like positionOffset has been renamed to positionOrigin. The base layer doc needs to be updated.

Obviously, I'm not 100% awake on the train to work....

Updated

File: src/layers/core/geojson-layer/geojson-layer.js [hunk 1]
```diff
@@ -82,17 +82,23 @@ export default class GeoJsonLayer extends CompositeLayer {
...
 
+    // Accessor props for underlying layers
+    const {getLineColor, getFillColor, getRadius,
+      getLineWidth, getElevation, updateTriggers} = this.props;
+
     // base layer props
     const {opacity, pickable, visible, projectionMode} = this.props;
 
+    // viewport props
+    const {positionOrigin, modelMatrix} = this.props;
```
Comment (Pessimistress) [inline src/layers/core/geojson-layer/geojson-layer.js]: projectionMode is a viewport prop also?

**Comment by shaojingli** [comment]
I agree with you @ibgreen. I'll get this PR landed and then create an issue to mitigate this problem

**Pull request merged**